de/main.po

```diff
@@ -1,815 +1,881 @@
+"Empfangen)."
 
 msgid "Auto Translate Users"
-msgstr ""
+msgstr "Automatisch Nutzer übersetzen"
 
 msgid ""
-"Automatically translates any new messages by specific users in the server "
-"(same channel)."
+"Automatically translates any new messages by specific users in the server (same "
+"channel)."
 msgstr ""
+"Übersetzt automatisch alle neuen Nachrichten von bestimmten Nutzern im Server "
+"(gleicher Kanal)."
 
 msgid "Translate user to channel's default language"
-msgstr ""
+msgstr "Übersetzt Nutzer zur Standardsprache des Kanals"
 
 msgid "Translate user to specific language"
-msgstr ""
+msgstr "Übersetzt Nutzer zu einer bestimmten Sprache"
 
 msgid "For example:"
-msgstr ""
+msgstr "Zum Beispiel:"
 
 msgid "Ban Users"
-msgstr ""
+msgstr "Nutzer sperren"
 
 msgid ""
 "This requires the bot to have **Manage Roles** and **Manage Nicknames** "
 "permissions."
 msgstr ""
+"Dies setzt voraus, dass der Bot über die Berechtigungen **Rollen verwalten** "
+"und **Nicknamen verwalten** verfügt."
 
 msgid "Allow Other Bots"
-msgstr ""
+msgstr "Andere Bots erlauben"
 
 msgid ""
 "To allow translation of other bots, simply create a new role called %s and "
 "assign it to the bots you want. (Use at your own risk)"
 msgstr ""
+"Um die Übersetzung anderer Bots zu ermöglichen, erstelle einfach eine neue "
+"Rolle namens %s und weise diese den gewünschten Bots zu. (Benutzung auf eigene "
+"Gefahr)"
 
 msgid "Stop Channel Automatic Translation"
-msgstr ""
+msgstr "Automatische Kanal-Übersetzung stoppen"
 
 msgid "Go to channel and write:"
-msgstr ""
+msgstr "Gehe zum Kanal und schreibe:"
 
 msgid "or"
-msgstr ""
+msgstr "oder"
 
 msgid "Alternatively, change the channel's mode"
-msgstr ""
+msgstr "Alternativ kann auch der Modus des Kanals geändert werden"
 
 msgid "(receives translated messages only)"
-msgstr ""
+msgstr "(empfängt nur übersetzte Nachrichten)"
 
 msgid "(sends translated messages only)"
-msgstr ""
+msgstr "(sendet nur übersetzte Nachrichten)"
 
 msgid "Stop User Automatic Translation"
-msgstr ""
+msgstr "Automatische Nutzer-Übersetzung stoppen"
 
 msgid "Ban User from using Translator"
-msgstr ""
+msgstr "Nutzer von der Verwendung des Translators ausschließen"
 
 msgid "Stop translating other Bots"
-msgstr ""
+msgstr "Übersetzen von anderen Bots stoppen"
 
 msgid "Simply remove the `%s` role from the other bots."
-msgstr ""
+msgstr "Entferne einfach die `%s` Rolle von den anderen Bots."
 
 msgid "Ignore specific messages"
-msgstr ""
+msgstr "Bestimmte Nachrichten ignorieren"
 
 msgid "Ignore specific words"
-msgstr ""
+msgstr "Bestimmte Wörter ignorieren"
 
 msgid "Miscellaneous Commands"
-msgstr ""
+msgstr "Verschiedene Befehle"
 
 msgid "Help"
-msgstr ""
+msgstr "Hilfe"
 
 msgid "Info"
-msgstr ""
+msgstr "Info"
 
 msgid "Links"
-msgstr ""
+msgstr "Links"
 
 msgid "Stats"
-msgstr ""
+msgstr "Statistiken"
 
 msgid "(bot invite link)"
-msgstr ""
+msgstr "(Bot Einladungslink)"
```

Comment:
`(Bot-Einladungslink)` would be better because we use a `-` for the other translations with the word `bot`.